.github/workflows/appimage.yml

```diff
@@ -48,31 +46,20 @@ jobs:
     - name: make clean
       run: |
         chown -R testbot:testbot .
-        sudo -u testbot bash -lc 'cd viam-cartographer && make clean'
```

Comment:
[q] Is removing this a benefit of the (TOOL_BIN) additions made to the Makefile?